### DISCUSSION TITLE
'ErrorBoundary' cannot be used as a JSX component

### DISCUSSION THREAD

**Discussion by pateljay1397** [webpack/webpack#16954] [Q&A]
**It happened after I updated my webpack version with "webpack": "^5.76.0".**

**Note**: I don't want to upgrade my react version to v18 at this moment.

I get typescript error when I use the default first example from your lib:

'ErrorBoundary' cannot be used as a JSX component, any chance you had seen that before?

![image](https://user-images.githubusercontent.com/54871480/230987595-2f24fb59-b28c-4a84-a8f5-18b7f39bc924.png)
**devDependencies**
- "react-error-boundary": "^3.1.4"

-  "@types/node": "^16.11.41",

-  "@types/react-dom": "^17.0.2",

-  "@types/react-redux": "^7.1.9",

-  "@types/react": "^17.0.2",

`
**dependencies**
-     "react-dom": "^17.0.2",

-     "react-redux": "^8.0.4",

-     "react": "^17.0.2",

import React from "react";
import ReactDOM from "react-dom";
import {ErrorBoundary} from "react-error-boundary";

import App from "./App";

const ErrorFallback = ({error}) => {
    return (
        <div>
            <h2>Something went wrong:</h2>
            <p>{error.toString()}</p>
        </div>
    );
};

ReactDOM.render(
    <ErrorBoundary FallbackComponent={ErrorFallback}>
        <App />
    </ErrorBoundary>,
    document.getElementById("root")
);

**Comment by alexander-akait** [comment]
Please create reproducible test repo

**Reply by alexander-akait** [reply]
Also it is `typescript` error, webpack does nothing with typescript

**Reply by pateljay1397** [reply]
It happened after upgrading the webpack version to 5.76.0

**Reply by alexander-akait** [reply]
No, it is not due webpack update, we even don't touch such things in https://github.com/webpack/webpack/releases/tag/v5.76.0, how you update webpack?
